ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -352,8 +352,8 @@ private void runScript(
       );
     } catch (final IOException e) {
       throw new KsqlException(
-          " Could not read statements from file: " + schemaFilePath + ". " + "Details: "
-          + e.getMessage(),
+          "Could not read statements from the provided script file " + schemaFilePath + ":\n"
+          + e + "\nMake sure the file exists and can be read by KSQL CLI.",
```

Comment:
I am not sure adding the new lines to the string is a good idea. It assumes too much about how the message is displayed. Let's drop the new lines and then we can merge this.

Reply:
@apurvam fixed